src/commands.ts

```diff
@@ -9,11 +9,13 @@ import statusbar from './statusbar';
 
 /* COMMANDS */
 
-async function refresh () {
+async function refresh ( silence? ) {
```

Comment:
"silence" doesn't doesn't sound super descriptive to me, can you replace `silence?` with `showNotification = true` maybe?